backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Controllers/LessonsController.cs

```diff
@@ -15,12 +16,31 @@ public LessonsController(ILessonsService lessonsService)
             _lessonsService = lessonsService;
         }
 
+        [HttpGet]
+        public async Task<ActionResult<ICollection<LessonDto>>> GetAllAsync()
+        {
+            var lessons = await _lessonsService.GetAllLessonsAsync();
+            return Ok(lessons);
+        }
+
         [HttpPost("filters")]
         public async Task<ActionResult<ICollection<LessonDto>>> GetAllAsync([FromBody] FiltersRequest filtersRequest)
         {
             var lessons = await _lessonsService.GetAllLessonsAsync(filtersRequest);
 
             return Ok(lessons);
         }
+
+        [HttpPost]
+        public async Task<ActionResult<LessonDto>> CreateAsync(NewLessonDto lessonDto)
+        {
+            if (!ModelState.IsValid)
```

Comment:
As far as I know we don't need it https://learn.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-7.0#automatic-http-400-responses